config/default.json

```diff
@@ -19,6 +19,7 @@
       { "period": "5 minutes", "limit": 0 }
     ]
   },
+  "jsonlogging": "false",
```

Comment:
@philbooth is this okay for 'default' and let prod set mozlog or should we set jsonlogging to false in local?

Reply:
commit `dev.json` with this `false`, set `true` to prod